### DISCUSSION TITLE
debug false stop livewire

### DISCUSSION THREAD

**Discussion by lucas-davi** [livewire/livewire#8005] [1. Help] [answered]
Hello!

I have a new project laravel with livewire installed.

After create counter component updated the AppServiceProvider file to:

```php
Livewire::setUpdateRoute(function ($handle) {
    return Route::post('/teste-app/public/livewire/update', $handle);
});

Livewire::setScriptRoute(function ($handle) {
    return Route::get('/teste-app/public/livewire/livewire.js', $handle);
});
```
This is working for me at first, but after set debug = false in .env file, livewire.js was no longer found.

Any idea? 

Sorry my bad english.

Thanks for help.

**Comment by joshhanley** [comment]
@lucas-davi Why are you overriding Livewire's routes? You generally don't need to, so I would recommend not doing it if possible.

**Reply by lucas-davi** [reply]
overwriting the routes was necessary because of my environment...

**Reply by joshhanley** [reply, marked as answer]
@lucas-davi hmm ok it's really bad practice to have `public/` in your routes though, as it leaves your root directory exposed so hackers can get to your `.env` file and get to your passwords and tokens.

But that's something you will need to decide if that is an ok risk.

To fix your problem, try just changing the script route definition (the update route one should be fine):

```php
Livewire::setScriptRoute(function ($handle) {
    return config('app.debug')
        ? Route::get('/teste-app/public/livewire/livewire.js', $handle)
        : Route::get('/teste-app/public/livewire/livewire.min.js', $handle);
});
```

We recently changed the default (see below for default), so this should hopefully give you the same functionality.

https://github.com/livewire/livewire/blob/14d59b01c7100de8018121b4b6ccd3c965fafe4f/src/Mechanisms/FrontendAssets/FrontendAssets.php#L20-L26

**Reply by lucas-davi** [reply]
Thanks for the warning about the `public` directory, my solution is really not good, I will work on it very soon.

The suggestion worked for me.

Thank you very much for the support.

**Reply by joshhanley** [reply]
@lucas-davi no worries! 🙂
